tests/ui/write_literal_2.stderr

```diff
@@ -68,9 +74,9 @@ LL | |         world!"
    |
 help: try this
    |
-LL ~         "some hello /
-LL ~         world!"
-   |
+LL -         "some {}",
+LL +         "some hello world!"
```

Comment:
Just realised this is a problem, the formatting of the literal should be preserved in the replacement, or escaped if that's not possible

![image](https://user-images.githubusercontent.com/1830331/159920936-9f83c4e8-940b-43ea-8d2c-4371990784d9.png)